numcosmo/math/ncm_stats_dist_nd_kde_studentt.c

```diff
@@ -1000,7 +1044,7 @@ _ncm_stats_dist_nd_kde_studentt_eval (NcmStatsDistNd *dnd, NcmVector *y)
       m2lnp_i += gsl_pow_2 (ncm_vector_fast_get (row_i, k) - ncm_vector_fast_get (self->v, k));
     }
 
-    e_i  = ncm_vector_get (self->weights, i) *_ncm_stats_dist_nd_kde_studentt_studentt(self, (m2lnp_i/ self->href2));
+    e_i  = ncm_vector_get (self->weights, i) * _ncm_stats_dist_nd_kde_studentt_studentt (self, (m2lnp_i / self->href2));
```

Comment:
There are some minor changes, such as this one, where I tried to maintain the right notation of the code.